travis-scripts/script_osx.sh

```diff
@@ -9,7 +9,7 @@ rm -f /Users/travis/build/fontforge/fontforge/.git/shallow
 set +e
 mkdir -p /tmp/fontforge-source-tree
 echo "brew build starting..."
-brew install --verbose fontforge --HEAD --with-x 
+brew install --verbose fontforge --HEAD --with-x --with-libspiro
```

Comment:
You included `libspiro` in the `brew install` arguments in the `before_install_osx.sh` script but you weren't building FontForge with `libspiro` enabled? Didn't seem deliberate, but will revert if it was.